eng/common/scripts/SemVer.ps1

```diff
@@ -179,6 +214,9 @@ class AzureEngSemanticVersion {
       "2.0.0-beta.10",
       "2.0.0-beta.2",
       "2.0.0-beta.1",
+      "2.0.0-alpha.20201221.5",
+      "2.0.0-alpha.20201221.2",
+      "2.0.0-alpha.20200920.1",
```

Comment:
Please add a case `2.0.0-alpha.20200920.10` and `2.0.0-alpha.20200920.01`

Reply:
.10 works, but `2.0.0-alpha.20200920.01` should not be an expected input (though because of the python differences, the regex will make it valid).

Similar to the above conversation about leading zeroes, perhaps we should just parse `BuildNumber` as string, and cast to int only when doing arithmetic?

Reply:
We can parse it as a string but it will not support in case we have more then 10 builds in a day.

Reply:
Having more than 10 builds a day is _very_ likely I would imagine.